python/lsst/ts/observing/block.py

```diff
@@ -189,6 +190,16 @@ class ObservingScript(BaseModel):
     parameters: dict[str, Any]
     """Parameters to pass to the observing script."""
 
+    def get_script_configuration(self) -> str:
+        """Get script configuration as a yaml string.
+
+        Returns
+        -------
+        str
```

Comment:
This is not numpydoc standard is it? Aren't we supposed to do:

```
Returns
--------
config : `str`
    Script configuration as a yaml string.
```
?

Reply:
I think it is... At least, when you are not returning a named parameter that is what the [numpy doc suggests](https://numpydoc.readthedocs.io/en/latest/format.html#returns).

The back quotes I recently learned is a [documenteer thing](https://developer.lsst.io/python/numpydoc.html#marking-up-parameter-names), so I will add it.

Reply:
Okay. DM dev guide agrees with me https://developer.lsst.io/python/numpydoc.html#py-docstring-returns

The dev guide explicitly says not to use the numpydoc suggestion because the sphinx docs don't render that properly.

Reply:
Ah! good to know! I'll implement it like this then!